weeklyChallenges04/ChallengesSet04.cs

```diff
@@ -1,52 +1,136 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 namespace weeklyChallenges04
 {
     public class ChallengesSet04
     {
         public int AddEvenSubtractOdd(int[] numbers)
         {
-            throw new NotImplementedException();
+            int numTotal = 0;
+            foreach (int num in numbers)
+            {
+                if (num % 2 == 0)
+                {
+                    numTotal += num;
+                }
+                else
+                {
+                    numTotal -= num;
+                }
```

Comment:
This was a decent use foreach loop but consider shortening the code by using a ternary operator:

```
 foreach (int num in numbers)
            {
                numTotal = (num % 2 == 0) ? numTotal + num : numTotal - num;
               
            }
                return numTotal;
```